scripts/release/go_update/go_obo_update.pl

```diff
@@ -372,79 +370,82 @@
 
 %obsolete = (%obsolete, %alternate, %pending_obsoletion);
 foreach my $obs_id ( sort keys %obsolete) {
-    
-    print FR "\|\[$gk_central_host/cgi-bin/instancebrowser\?DB\=$opt_db\&ID\=$obsolete{$obs_id}\& $total_name{$obs_id}\]\n";
-    print FR "\|\[http://www.ebi.ac.uk/ego/GTerm\?id\=GO:$obs_id GO:$obs_id\]\n";
-            
-    my $action;
-    my @terms = ();
-    if ( $replaced{$obs_id}[0] ) {
-        $action = 'replace by:';
-        @terms = @{ $replaced{$obs_id} };
-    } elsif ( $consider{$obs_id}[0]) {
-    	$action = 'consider:';
-    	@terms = @{ $consider{$obs_id}};
-    } elsif ( $alternate{$obs_id}) {
-    	$action = 'alternate id -- no action required';
-	
-	# Move all referrers to the primary GO term
-	my $alternate_acc = $obs_id;
-	my $alt_ac = $dba->fetch_instance_by_db_id( $alternate{$alternate_acc} );
-	my $primary = $dba->fetch_instance_by_db_id($altacc_to_primaryacc{$alternate_acc})->[0];
-	
-	foreach my $sdi ( @{$alt_ac} ) {
-	    $sdi->inflate();
-	    my $alternate_accession = $sdi->accession->[0];
-	    my $ref   = $dba->fetch_referer_by_instance($sdi);
-	    foreach my $go_ref ( @{$ref} ) {
-	        $go_ref->inflate();
-		
-		# Skip referrers to other GO instances
-	        my $dbr = $go_ref->referenceDatabase->[0]; 
-	        next unless (!$dbr || $dbr->db_id != 1 );
-	        
-		
-		foreach my $attribute ($go_ref->list_attributes()) {
-		    my $attribute_value = $go_ref->$attribute->[0];
-		    next unless $attribute_value;
-		    
-		    my $accession;
-		    
-		    eval {
-			$accession = $attribute_value->accession->[0];
-		    };
-		    
-		    next unless $accession;
-		    
-		    # If the attribute's value is a GO instance with an
-		    # accession matching the alternate term, give the
-		    # primary GO instance as the attribute value instead.
-		    # This, in effect, moves the referrers from the alternate
-		    # GO term to the primary GO term
-		    if ($accession == $alternate_accession) {
-			$go_ref->$attribute($primary);
-			$dba->update($go_ref);
-		    }
+
+	print FR "\|\[$gk_central_host/cgi-bin/instancebrowser\?DB\=$opt_db\&ID\=$obsolete{$obs_id}\& $total_name{$obs_id}\]\n";
+	print FR "\|\[http://www.ebi.ac.uk/ego/GTerm\?id\=GO:$obs_id GO:$obs_id\]\n";
+
+	my $action;
+	my @terms = ();
+	if ( $replaced{$obs_id}[0] ) {
+		$action = 'replace by:';
+		@terms = @{ $replaced{$obs_id} };
+	}
+	elsif ( $consider{$obs_id}[0]) {
+		$action = 'consider:';
+		@terms = @{ $consider{$obs_id}};
+	}
+	elsif ( $alternate{$obs_id}) {
+		$action = 'alternate id -- no action required';
+
+		# Move all referrers to the primary GO term
+		my $alternate_acc = $obs_id;
+		my $alt_ac = $dba->fetch_instance_by_db_id( $alternate{$alternate_acc} );
+		my $primary = $dba->fetch_instance_by_db_id($altacc_to_primaryacc{$alternate_acc})->[0];
+
+		foreach my $sdi ( @{$alt_ac} ) {
```

Comment:
This $sdi variable name is quite mysterious, though I know that it is from previous iterations

Reply:
The code is iterating through alternate accessions. I can't figure out what `$sdi` might refer to. something-something-_Instance_?

Reply:
Yeah, I never figured it out either.  It is just a GO instance I believe.